### ISSUE TITLE
NUnitRetry.SpecflowPlugin doesn't seem to be working in Jenkins pipeline

### ISSUE THREAD

**Issue by zrcharbo** [farum12/NUnitRetry.SpecFlowPlugin#3]
I am using the plugin in my specflow project that we have running in our jenkins pipeline. The retry seems to be working when I run the tests locally but it doesn't work when the tests are running in jenkins. Is there some sort of configuration that needs to be done or is this a known defect?

**Comment by farum12** [comment]
@zrcharbo Nope, unfortunately, you're the first one to report that issue. I haven't tried this plugin against Jenkins. 
Please ensure:
- that you're using the newest NuGet, 
- specflow.json is copied properly to Debug/netXXX or Release/netXXX directory
- specflow.json contains the NRetrySettings section

Is there any possibility to send me some logs/reports?
